x-pack/plugins/triggers_actions_ui/public/application/components/builtin_action_types/servicenow/servicenow_itsm_params.tsx

```diff
@@ -169,14 +187,55 @@ const ServiceNowParamsFields: React.FunctionComponent<
               hasNoInitialSelection
               isLoading={isLoadingChoices}
               disabled={isLoadingChoices}
-              options={options.impact}
+              options={impactOptions}
               value={incident.impact ?? ''}
               onChange={(e) => editSubActionProperty('impact', e.target.value)}
             />
           </EuiFormRow>
         </EuiFlexItem>
       </EuiFlexGroup>
       <EuiSpacer size="m" />
+      <EuiFlexGroup>
+        <EuiFlexItem>
+          <EuiFormRow fullWidth label={i18n.CATEGORY_LABEL}>
+            <EuiSelect
+              fullWidth
+              data-test-subj="categorySelect"
+              hasNoInitialSelection
+              isLoading={isLoadingChoices}
+              disabled={isLoadingChoices}
+              options={categoryOptions}
+              value={incident.category ?? undefined}
+              onChange={(e) => {
```

Comment:
I'd do a useCallback here since you have the `incident` value

Reply:
There is no difference if we use the incident or not. The callback function will always be a new reference for the prop.